livekit_models.proto

```diff
@@ -57,6 +57,11 @@ enum VideoCodec {
   VP8 = 4;
 }
 
+enum ImageCodec {
+  DEFAULT_IC = 0;
```

Comment:
IC prefix is preferred

IC_DEFAULT
IC_JPEG

Reply:
Agreed that a prefix is cleaner. This is inconsistent with the current audio and video codec definitions though:

```
enum AudioCodec {                 
  DEFAULT_AC = 0;                       
  OPUS = 1;                                          
  AAC = 2;          
}                                                       
                           
enum VideoCodec {           
  DEFAULT_VC = 0;      
  H264_BASELINE = 1;                                                   
  H264_MAIN = 2;                          
  H264_HIGH = 3;          
  VP8 = 4;                                                                       
}    
```

Do we mind?

Reply:
I think @boks1971 proposed prefixes after some of these are already in-use. So I think we can keep the newly adopted patterns without going back and changing everything else; since doing so may cause some breakage.